loops.js

```diff
@@ -49,4 +61,3 @@
 // >> 7
 // >> 9
 // ==========================================
-
```

Comment:
Ik zie de uitwerking van de bonusopdracht niet.
Ik had deze als volgt uitgewerkt: 
for (let i = 0; i < 9; i++) {
    if (i > 2 && i < 6) {
        console.log("> " + i);
    }
    else if (i > 5 && i < 9) {
        console.log(">> " + i);
    }
    else { console.log(i);
    }
}